Mathlib/Data/Fintype/Card.lean

```diff
@@ -7,6 +7,7 @@ import Mathlib.Data.Fintype.Basic
 import Mathlib.Data.Finset.Card
 import Mathlib.Data.List.NodupEquivFin
 import Mathlib.Tactic.Positivity
+import Mathlib.Tactic.WLOG
```

Comment:
This added import is technically not needed, but I added it for "include what you use" style.

Reply:
I will drop it because unnecessary imports are a pain to deal with when trying to split files. Hope you don't mind too much.
```suggestion
```